map/map_tf_generator/src/map_tf_generator_node.cpp

```diff
@@ -54,16 +58,16 @@ class MapTFGeneratorNode : public rclcpp::Node
     PointCloud clouds;
     pcl::fromROSMsg<pcl::PointXYZ>(*clouds_ros, clouds);
 
-    const unsigned int sum = clouds.points.size();
+    const std::vector<int> indices = UniformRandom(clouds.size(), N_SAMPLES);
```

Comment:
```suggestion
    const std::vector<size_t> indices = UniformRandom(clouds.size(), N_SAMPLES);
```

Reply:
Is there a specific reason to use size_t instead of int?

Reply:
If it represents indices and won't be negative, I believe `size_t` is appropriate.
`clouds.size()` returns `size_t`.

Reply:
Understood!

Reply:
Resolved in https://github.com/autowarefoundation/autoware.universe/pull/890/commits/9ef92be63986d219e34cc0aeb6c628746be5296f